Vishnu/Assignment3.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+<body>
+<form action="" method="post">
+<h1>Register</h1>
+<p>
+    First Name <input type="text" name="FirstName" required autocomplete="given-name">
+    Last Name <input type="text" name="LastName" required autocomplete="family-name">
+</p>
+<p>
+    Gender: Male: <input type="radio" name="Gender" value="male"/>
+    Female: <input type="radio" name="Gender" value="female"/>
+    Others: <input type="radio" name="Gender" value="others"/>
+</p>
+<p>
+Email: <input type="email" name="Email" id="email" required autocomplete="email">
+Password: <input type="password" name="pwd" min="5" max="15" required autocomplete="current-password">
```

Comment:
Wrong `autocomplete` attribute. Should be for a registration form, not login. So should be new and not current as there is no current.